pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/widgets/cluster-class-info/cluster-class-info.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit } from '@angular/core';
```

Comment:
nitpick, please put '// Angular imports' comment on line 1 and then '// App imports' comment on line 3